mne/preprocessing/_css.py

```diff
@@ -0,0 +1,174 @@
+
+
+import numpy as np
+import mne
+import matplotlib.pyplot as plt
+
+
+def _css(mag_data, grad_data, filter_data, r=6):
```

Comment:
r -> n_proj?

single letter variable names are not usually encouraged because they are hard to Ctrl + F